content/test/integration/check-failed-deployments.spec.ts

```diff
@@ -0,0 +1,263 @@
+import fetch, { Response } from "node-fetch"
+import { FailedDeployment } from "@katalyst/content/service/errors/FailedDeploymentsManager"
+import { EntityType, EntityId, Entity, Pointer } from "@katalyst/content/service/Entity"
+import { AuditInfo } from "@katalyst/content/service/audit/Audit"
+import { EthAddress, AuthChain, ValidationResult } from "dcl-crypto"
+import { Timestamp } from "@katalyst/content/service/time/TimeSorting"
+import { AccessCheckerImpl } from "@katalyst/content/service/access/AccessCheckerImpl"
+import { ContentAuthenticator } from "@katalyst/content/service/auth/Authenticator"
+import { DEFAULT_DCL_PARCEL_ACCESS_URL } from "@katalyst/content/Environment"
+import fs from 'fs';
+import { httpProviderForNetwork } from "../../../contracts/utils"
+
+describe("Failed Deployments validations.", () => {
```

Comment:
Maybe we could separate from the automatic tests? Maybe move them to a new folder called tools/tooling or something similar?